tests/e2e/util.go

```diff
@@ -5746,3 +5751,14 @@ func waitAndGetContainerID(sshClientConfig *ssh.ClientConfig, k8sMasterIP string
 	}
 	return containerId, waitErr
 }
+
+//startVCServiceWait4VPs starts SPS and waits for all VPs to come online
+func startVCServiceWait4VPs(ctx context.Context, vcAddress string, service string, isSpsStopped *bool) {
```

Comment:
Since this code is not specific to SPS . We can say  'isServiceStopped' instead of  'isSpsStopped'
Also update the comment with services instead of SPS

Reply:
My bad! Fixed it now.